lib/sequence.js

```diff
@@ -61,28 +63,31 @@ Sequence = function(schema, options) {
  *
  * @static
  */
-Sequence.getInstance = function(schema, options) {
+Sequence.getInstance = function (schema, options) {
     var sequence = new Sequence(schema, options),
-        id = sequence.getId();
+        id = sequence.getId(),
+        existsSequence = sequenceArchive.existsSequence(id);
 
-    if(sequenceArchive.existsSequence(id)){
-        throw new Error('Counter already defined for field "'+id+'"');
+    sequence.enable.call(sequence);
```

Comment:
This shouldn't be needed. Probably

```js
sequence.enable()
```

is enough, isn't it?

Reply:
Maybe it's enough, my bad, I usually call methods like this way for "safety" (ensure the "this" in the other method is referrencing correcty the context of the instance.). i will test run and test it, in case of succes I will remove it.

Reply:
:smile:  Fear of the `this`, like Iron maiden would say!